File: year2020/month09/day0925/ms_0925.cpp
#include <iostream>

using namespace std;

int recursionPow(int n);

int main() {
	int n;

	cin >> n;

	if (1 <= n && n <= 20)
		cout << recursionPow(n) << endl;

	return 0;
}

int recursionPow(int n) {
	if (n == 0)
		return 1;
	else
		return 2 * recursionPow(n - 1);
}

File: year2020/month09/day0925/ms_0925.cs
using System;

namespace algorithm_65
{
    class ms_0925
    {
        static int RecursionPow(int n) 
        {
            if(n == 0)
                return 1;
            else 
                return 2 * RecursionPow(n-1);
        }

        static void Main(string[] args)
        {
            int n = Convert.ToInt32(Console.ReadLine());

            if(1 <= n && n <= 20)
                Console.WriteLine(RecursionPow(n));
        }
    }
}
